packages/server/lib/browsers/electron.js

```diff
@@ -307,9 +307,16 @@ module.exports = {
           id: downloadItem.getETag(),
         })
       })
-    })
+    }
+
+    const { session } = win.webContents
+
+    session.on('will-download', onWillDownload)
+
+    // avoid adding redundant `will-download` handlers if session is reused for next spec
+    win.on('closed', () => session.removeListener('will-download', onWillDownload))
```

Comment:
apparently the session is reused between webContents, because this eventually emits the same max listeners warning after 12 or so specfiles have launched Electron